chalicelib/action.py

```diff
@@ -145,7 +145,29 @@ def _delete_action(self):
         return ""
 
     def _edit_action(self):
-        return self.send_response(message="Edit not implemented yet")
+        """
+        Edit event in timereport for user.
+        If no arguments supplied it will try to edit today.
+        """
+        
+        date = datetime.now().strftime("%Y-%m-%d")
+
+        try:
+            if self.params[1] != "today":
+                date = self.params[1]
+        except IndexError:
+            log.debug(f"Didn't get any params. Setting date to {date}")
+
+        event_to_edit = self._get_events(date_str=date)
+        log.debug(f"Event to edit is: {event_to_edit}")
+        
+        if event_to_edit == '[]':
```

Comment:
What happens if event_to_edit is None or empty string?

Reply:
I see your point. Should I set e default value or should I also check for `None`

Reply:
Check using something like this:

if not event_to_list:
  print "Nothing here"

Handles None, "" and [] all in one.

Reply:
Yeah the problem is that '[]' isn't none. But yeah I guess I should decode the JSON first and then it will be an empty list == None

Reply:
@stefan-julin I've pushed a commit to change this behaviour. Please check